pkg/reconciler/v1alpha1/taskrun/resources/pod_test.go

```diff
@@ -111,50 +109,17 @@ func TestMakePod(t *testing.T) {
 			},
 			Volumes: implicitVolumes,
 		},
-	}, {
-		desc: "gcs-source-with-targetPath",
-		b: v1alpha1.BuildSpec{
-			Source: &v1alpha1.SourceSpec{
-				Name: "gcs-foo-bar",
-				GCS: &v1alpha1.GCSSourceSpec{
-					Type:     v1alpha1.GCSManifest,
-					Location: "gs://foo/bar",
-				},
-				TargetPath: "path/foo",
```

Comment:
do we still need `TargetPath` covered in this test, using resources (https://github.com/tektoncd/pipeline/blob/3e8b177fbeb39126a9c9de121008db571ef45bfa/docs/tasks.md#controlling-where-resources-are-mounted) ? (im not sure myself!)

Reply:
I think it's already covered with resources. This code was testing something we never set as is anymore I think :angel:

Reply:
kk coo :D